api/machine/machine.proto

```diff
@@ -171,12 +171,33 @@ message RestartEvent {
   int64 cmd = 1;
 };
 
+// MachineConfigErrorEvent means some machine config error.
+message MachineConfigErrorEvent {
```

Comment:
I think we should add new events so that they can be handled by `talosctl events` - it has some code to unpack the events for display

Reply:
two new kinds, right?

Reply:
three new kinds now ;)